.travis.yml

```diff
@@ -6,11 +7,12 @@ before_install:
   - echo "options(repos = c(CRAN='http://cran.r-project.org'))" > .Rprofile
 
 before_script:
-  - Rscript -e "install.packages('knitr', repos='http://cran.cnr.Berkeley.edu/')"
-  - Rscript -e "install.packages('rmarkdown', repos='http://cran.cnr.Berkeley.edu/')"
+  - Rscript -e "install.packages('knitr')"
+  - Rscript -e "install.packages('rmarkdown')"
 
 r_github_packages:
   - jimhester/covr
+  - hadley/testthat
```

Comment:
I added this before any of the tests were running. Before you let me know that the test_check () line was commented out in testthat.r I was trying different things to try to get tests to run. This should be removed.